waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -422,23 +445,24 @@ proc getCurrentEpoch*(): Epoch =
 
 proc compare*(e1, e2: Epoch): int64 =
   ## returns the difference between the two rln `Epoch`s `e1` and `e2`
-  ## i.e., e1 - e2 
-  
+  ## i.e., e1 - e2
+
   # convert epochs to their corresponding unsigned numerical values
-  let 
+  let
     epoch1 = fromEpoch(e1)
     epoch2 = fromEpoch(e2)
   return int64(epoch1) - int64(epoch2)
 
-proc validateMessage*(rlnPeer: WakuRLNRelay, msg: WakuMessage, timeOption: Option[float64] = none(float64)): MessageValidationResult =
+proc validateMessage*(rlnPeer: WakuRLNRelay, msg: WakuMessage,
+    timeOption: Option[float64] = none(float64)): MessageValidationResult =
   ## validate the supplied `msg` based on the waku-rln-relay routing protocol i.e.,
   ## the `msg`'s epoch is within MAX_EPOCH_GAP of the current epoch
   ## the `msg` has valid rate limit proof
   ## the `msg` does not violate the rate limit
-  ## `timeOption` indicates Unix epoch time (fractional part holds sub-seconds) 
+  ## `timeOption` indicates Unix epoch time (fractional part holds sub-seconds)
```

Comment:
Why times in RLN Relay are intended as floats? Would it better to align everything to int64 or there are some particular reason to keep them as floats?

Reply:
Fair enough, tracking issue https://github.com/status-im/nim-waku/issues/946